### DISCUSSION TITLE
链接？

### DISCUSSION THREAD

**Discussion by Qzzz66** [Coolapk-UWP/Coolapk-Lite#29] [Q&A] [answered]
下载链接里没有东西啊，空的

**Comment by wherewhere** [comment, marked as answer]
[![Build Status](https://dev.azure.com/wherewhere/Coolapk-UWP/_apis/build/status/Coolapk-UWP.Coolapk-Lite?branchName=master)](https://dev.azure.com/wherewhere/Coolapk-UWP/_build/latest?definitionId=5&branchName=master)

**Reply by Qzzz66** [reply]
这个网站怎么用啊 不太会 看着和github挺像，不知道怎么下载
